doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -607,344 +797,50 @@ Response code:
 
 + Response 204
 
+## Standard queryContext [/v2/queryContext]
 
-## Context Entity Subscriptions [/v2/entities/{entityId}/subscriptions]
```

Comment:
By the moment, I have removed the not-yet-discussed operations in the reference document. We already have these examples in examples .apib anyway.